qa/valgrind.supp

```diff
@@ -198,20 +198,6 @@
 	fun:*set_flush_on_exit*
 	...
 }
-{
-	libleveldb: ignore all static leveldb leaks
-	Memcheck:Leak
-	...
-	fun:*leveldb*
-	...
-}
-{
-	libleveldb: ignore all dynamic libleveldb leaks
-	Memcheck:Leak
-	...
-	obj:*libleveldb.so*
-	...
-}
```

Comment:
ACK. Adding this comment to grab attention if something somehow pops up in testing.